Aptoide/aptoide-client-v8
# [MOB-130] Rate and reviews

**What does this PR do?**

   This PR aims at fixing the some small rate and reviews view issues, like the the thumbs up and thumbs down buttons and the post a comment button.

**Database changed?**

 No

**Where should the reviewer start?**

- [ ] RateAndReviewCommentWidget.java

**How should this be manually tested?**

  Open the reviews of an app and check if everything is correct (buttons and colors).

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-130](https://aptoide.atlassian.net/browse/MOB-130)
[MOB-117](https://aptoide.atlassian.net/browse/MOB-117)
**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
